### PR TITLE
python-zipp: add new package

### PR DESCRIPTION
**Pull request by ja-pa**
Maintainer: me
Compile tested: Turris Omnia (TOS5), OpenWrt master
Run tested: Turris Omnia (TOS5), OpenWrt master

Description:
This PR is based on #8562 . It splits new packages to individuals PR.

### CODE REVIEW

File: lang/python/python-zipp/Makefile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-zipp
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=Zipfile object wrapper
+  URL:=https://github.com/jaraco/zipp
+  DEPENDS:=+python3-light
```
Comment (jefferyto) [inline lang/python/python-zipp/Makefile]: [v0.6.0](https://github.com/jaraco/zipp/blob/master/CHANGES.rst#v060) added a [dependency](https://github.com/jaraco/zipp/blob/v0.6.0/setup.cfg#L24-L25) on more_itertools.

File: lang/python/python-zipp/Makefile [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-zipp
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=Zipfile object wrapper
+  URL:=https://github.com/jaraco/zipp
+  DEPENDS:=+python3-light +python-more_itertools
```
Comment (jefferyto) [inline lang/python/python-zipp/Makefile]:
```suggestion
  DEPENDS:=+python3-light +python-more-itertools
```

Comment (ja-pa) [reply lang/python/python-zipp/Makefile]: Fixed thanks

**Comment by hnyman** [comment]
Sounds like the python-more-itertools package Makefile actually defines a python3 package and this zipp depends on a non-existing python2 package, which causes error in config:

```
perus@ub1910:/Openwrt/r7800$ make defconfig
WARNING: Makefile 'package/feeds/packages/python-zipp/Makefile' has a dependency on 'python-more-itertools', which does not exist
#
# configuration written to .config
#
```
Reference to:
https://github.com/openwrt/packages/pull/10407/files#diff-2ae74290fa838eb98ea375a59c90166cR25

**Comment by BKPepe** [comment]
Thanks. Fixed by commit https://github.com/openwrt/packages/commit/426ed75dbeec4448ee2cec16bf7c78ee7b3528dc

**Pull request merged**